### PR TITLE
Added TextDriver with getContent method #216

### PR DESCRIPTION
**Pull request by daniel-martic-sociomantic**
Added TextDriver with getContent method #216

### CODE REVIEW

**Comment by conor-cafferkey-sociomantic** [comment]
hey @ido-flaishon-sociomantic, could you give this a quick review whenever you have a chance

File: src/Text/tests.jsx [hunk 1]
```diff
@@ -26,3 +26,46 @@ describe( 'Text', () =>
...
+    {
+        wrapper = mount( <Text /> );
+        driver  = wrapper.driver();
+    } );
+
+    describe( 'getContent', () =>
+    {
+        it( 'should return the content set by text prop', () =>
+        {
+            const text = 'the quick brown fox jumps over the lazy dog';
```
Comment (ido-flaishon-sociomantic) [inline src/Text/tests.jsx]: You can be more original next time.

File: src/Text/tests.jsx [hunk 2]
```diff
@@ -26,3 +26,46 @@ describe( 'Text', () =>
...
+            wrapper.setProps( { text } );
+            expect( driver.getContent() ).to.equal( text );
+        } );
+
+        it( 'should return the content set by children prop', () =>
+        {
+            const text = 'the quick brown fox jumps over the lazy dog';
+            const children = <div>{ text }</div>;
+
+            wrapper.setProps( { children } );
```
Comment (ido-flaishon-sociomantic) [inline src/Text/tests.jsx]: Is this completely equivalent to mounting straight on:
```
<Text>
     <div>I can be original</div>
</Text>
```
?

Cause this what will happen most likely here and there.

File: src/Text/tests.jsx (hunk 1 above)
Comment (conor-cafferkey-sociomantic) [reply src/Text/tests.jsx]: 😜

File: src/Text/tests.jsx (hunk 2 above)
Comment (conor-cafferkey-sociomantic) [reply src/Text/tests.jsx]: yep it’s the same

**Pull request merged**